client/dashboard/sites/add-new-site/menu-item.tsx

```diff
@@ -0,0 +1,42 @@
+import {
+	__experimentalVStack as VStack,
+	__experimentalHStack as HStack,
+	__experimentalText as Text,
+	Button,
+} from '@wordpress/components';
+import { Icon } from '@wordpress/icons';
+import React from 'react';
+
+type Props = {
+	children?: JSX.Element;
+	description: string;
+	title: string;
+	icon: JSX.Element;
+};
+
+function MenuItem( {
+	children,
+	description,
+	title,
+	icon,
+	...buttonProps
+}: Props & React.ComponentProps< typeof Button > ) {
+	return (
+		<Button { ...buttonProps } label={ title } style={ { height: 'auto', textAlign: 'left' } }>
```

Comment:
I think the tooltip is redundant, since it repeats the menu item title:

![Screenshot 2025-05-14 at 17 33 35](https://github.com/user-attachments/assets/909cec00-901a-4109-8e4e-cd63fafa5d17)

The aria-label is useful, though, so perhaps we should specifically disable `showTooltip`?